### DISCUSSION TITLE
I just installed it via Docker, but there is no option to enable WhatsApp.

### DISCUSSION THREAD

**Discussion by GlauberF** [Tiledesk/tiledesk#24] [Q&A]
I need to know how to enable WhatsApp, as I will use it within my company to receive support via WhatsApp, where it can be automated, but can be attended to by one of our employees.
My other question, messages sent on my WhatsApp, I can see within the chat, so that my employees/attendants can mark them as completed/archived.

Appears only when installing and configuring Telegram
![Captura de tela de 2024-04-30 14-31-14](https://github.com/Tiledesk/tiledesk/assets/11314585/8471617e-f5ee-4235-81c3-28156effb026)

**Comment by sponzillo** [comment]
Hi , Whatsapp integration is still not released as a public Github project. We'll provide you with an update when we will release the Whatsapp integration.

Bye, Andrea

**Reply by GlauberF** [reply]
Forecast for this?
Because for me, the project only makes sense with this integration with WhatsApp.

**Reply by nicobts** [reply]
For me as well. Very important to enable integrations to IM platforms.
Whatsapp most important
